pep-0688.rst

```diff
@@ -0,0 +1,258 @@
+PEP: 688
+Title: Making the buffer protocol accessible in Python
+Author: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 23-Apr-2022
+Python-Version: 3.12
+
+
+Abstract
+========
+
+This PEP proposes a mechanism for Python code to inspect whether a
+type implements the C-level buffer protocol. This allows type
+checkers to check for objects that implement the protocol.
```

Comment:
Thanks, I'll go with the first one. I don't think the second sentence is necessary.

I'm applying the changes locally and will push them when I'm done, in case I want to make more substantive edits.